tutorials/02a_RunningREMINDLocally.md

```diff
@@ -1,10 +1,146 @@
 # Running REMIND locally
-Mika Pflüger (mika.pflueger@pik-potsdam.de)
+Mika Pflüger (mika.pflueger@pik-potsdam.de), Tonn Rüter (tonn.rueter@pik-potsdam.de)
 
-*Note:* This tutorial is a work in progress, not all steps needed are described, yet.
+*Note:* This approach of running REMIND is a work in progress, however successful REMIND runs on local machines have been conducted using this tutorial.
+
+## System Requirements
+
+- Installed python version must be `3.10.X` or older. To check your python version run `python --version` in a terminal of your choice
+- GAMS must be installed and the path to GAMS applications needs to be in your PATH environment variable
+- R version must be at least `4.2.X`. To check your R version run `R --version` in a terminal of your choice
+- On Windows we recommend to use PowerShell rather than the standard command prompt
+
+## Use Local Input Data
```

Comment:
Should be structured as follows

## Getting Input Data

### Use Local Input Data
...

### Download Input Data Automatically
....